src/sentry/static/sentry/app/components/panels/panelItem.jsx

```diff
@@ -1,18 +1,21 @@
 import {Flex} from 'grid-emotion';
 import PropTypes from 'prop-types';
 import styled from 'react-emotion';
+import isPropValid from '@emotion/is-prop-valid';
 
-const PanelItem = styled(Flex)`
+const PanelItem = styled(Flex, {shouldForwardProp: isPropValid})`
   border-bottom: 1px solid ${p => p.theme.borderLight};
 
   &:last-child {
-    border: 0;
+    border: ${p => (p.forceBorder ? null : 0)};
```

Comment:
Should the entire `border` property be conditional?

Reply:
unfortunately the linter chews it up. 

The compiler omits invalid css lines, so returning `null` will cause this to only show up for classes where `forceBorder` is true.